src/software/test_util/test_util.cpp

```diff
@@ -8,8 +8,9 @@
 
 namespace TestUtil
 {
-    World createBlankTestingWorld(Field field)
+    World createBlankTestingWorld(std::unique_ptr<TbotsProto::Field> field_proto)
     {
+        Field field        = Field(*field_proto);
```

Comment:
not quite what I meant but it's fine, it'll be cleaned up later

Reply:
changed it